dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -59,6 +59,9 @@ public class Excel : SupportedDataFrameFormat {
         DefaultReadExcelMethod(pathRepresentation)
 }
 
+private const val MESSAGE_1_1 = "Will be ERROR in 1.1."
```

Comment:
HIDDEN >= ERROR already. You can remove this message or say that they are removed in 1.1